etc/eod-config/eod-config.yml

```diff
@@ -261,6 +262,33 @@ eod_subnet_demo_vxlan:
   allocation_pool_start: "10.1.0.2"
   allocation_pool_end: "10.1.0.254"
 
+# EoD demo provider VLAN network name.
+eod_network_demo_provider_name: "demo-provider"
+
+# EoD demo provider VLAN
+eod_network_demo_provider:
+  name: "{{ eod_network_demo_provider_name }}"
+  project: "{{ eod_demo_project.name }}"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 100
+  shared: false
+  # Subnet configuration.
+  subnets:
+    - "{{ eod_subnet_demo_provider }}"
+
+# EoD demo provider VLAN subnet
+eod_subnet_demo_provider:
+  name: "{{ eod_network_demo_provider_name }}"
+  project: "{{ eod_demo_project.name }}"
+  cidr: "10.100.0.0/16"
+  gateway_ip: "10.100.0.1"
+  allocation_pool_start: "10.100.1.0"
+  allocation_pool_end: "10.100.99.255"
+  host_routes:
+    - destination: "10.66.0.0/16"
+      nexthop: "10.100.0.2"
```

Comment:
Where is this router? On the 1G switch?

Reply:
Yes - I have a list of things to catch up on with you and this is one... there's a question over where these networks get defined and where the routes for them are defined.